app/views/users/show.erb

```diff
@@ -0,0 +1,37 @@
+<h1> Accout of: <%= @user.username %> </h1>
+
+<p>***************************************************************************************************************************************************************************************************************************************************************************</p>
+
+<%= erb :"partials/_logged_in_current_user", layout: false, locals: {user: @user} %>
+
+<p>***************************************************************************************************************************************************************************************************************************************************************************</p>
+
+<div>
+  <h3> Your Suggestions </h3>
+  <% @suggestions.each do |suggestion| %>
+    <% if own_suggetion?(suggestion) %>
+      <h4><%= suggestion.title %> </h4>
+      <p><%= suggestion.body %> </p>
+        <a href="/suggestions/<%= suggestion.id %>/edit">Edit</a>
+        <%= erb :"suggestions/_delete", layout: false, locals: {suggestion: suggestion} %>
+    <% end %>
+  <% end %>
+</div>
+
+<p>***************************************************************************************************************************************************************************************************************************************************************************</p>
+
+<div>
+  <h3> Your Up Votes </h3>
+  <% @up_votes.map do |up_vote| %>
```

Comment:
no need for `map` since nothing is being done with its return value. use `each` here